node/src/proxy_server/mod.rs

```diff
@@ -2608,17 +2979,45 @@ mod tests {
 
     #[test]
     #[should_panic(
-        expected = "Each route must demand an exit service, but this route has no such demand: [Routing(0x726F7574696E675F6B65795F31, \
-    Wallet { kind: Address(0x00000000726f7574696e675f77616c6c65745f31) }, RatePack { routing_byte_rate: 9, \
-    routing_service_rate: 208, exit_byte_rate: 11, exit_service_rate: 408 })]"
+        expected = "RouteResultMessage Handler: stream key: +dKB2Lsh3ET2TS/J/cexaanFQz4 not found within dns_failure_retries"
     )]
-    fn proxy_server_panics_when_exit_services_are_not_requested_in_non_zero_hop_mode() {
-        let expected_services = vec![ExpectedService::Routing(
-            PublicKey::from(&b"routing_key_1"[..]),
-            make_wallet("routing_wallet_1"),
-            rate_pack(8),
-        )];
-
+    fn route_result_message_handler_panics_when_dns_retries_hashmap_doesnt_contain_a_stream_key() {
```

Comment:
I think this concern isn't as critical as it may appear. Here are the reasons behind this statement - https://github.com/MASQ-Project/Node/pull/326#discussion_r1305364710